presto-main/src/main/java/com/facebook/presto/sql/planner/sanity/ValidateDependenciesChecker.java

```diff
@@ -79,40 +80,40 @@
     @Override
     public void validate(PlanNode plan, Session session, Metadata metadata, SqlParser sqlParser, Map<Symbol, Type> types)
     {
-        plan.accept(new Visitor(), null);
+        plan.accept(new Visitor(), ImmutableList.of());
     }
 
     private static class Visitor
-            extends PlanVisitor<Void, Void>
+            extends PlanVisitor<List<Symbol>, Void>
     {
         private final Map<PlanNodeId, PlanNode> nodesById = new HashMap<>();
 
         @Override
-        protected Void visitPlan(PlanNode node, Void context)
+        protected Void visitPlan(PlanNode node, List<Symbol> correlation)
```

Comment:
I'd change the term `correlations` to `boundSymbols` or `boundVariables` (as in "free" vs "bound" variables in programming languages)